### PR TITLE
nx: add page

### PR DESCRIPTION
**Pull request by Locour**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/nx.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# nx
+
+> CLI utility for managing `nx` workspaces.
+> More information: <https://nx.dev/l/r/getting-started/nx-cli>.
+
+- Build a single project:
```
Comment (navarroaxel) [inline pages/common/nx.md:6]:
```suggestion
- Build a specific project:
```

File: pages/common/nx.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`nx build {{project}}`
+
+- Test a single project:
```
Comment (navarroaxel) [inline pages/common/nx.md:10]:
```suggestion
- Test a specific project:
```

File: pages/common/nx.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`nx test {{project}}`
+
+- Execute a target on a single project:
```
Comment (navarroaxel) [inline pages/common/nx.md:14]:
```suggestion
- Execute a target on a specific project:
```

**Review by marchersimon — APPROVED** [review]
Looks good after ＠navarroaxel's comment has been addressed.
Thanks

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**